src/compiler/diagnosticMessages.json

```diff
@@ -5294,7 +5442,7 @@
         "category": "Error",
         "code": 6258
     },
-      "Found 1 error in {0}": {
+    "Found 1 error in {0}": {

```

Comment:
Curious why it isn't formatted previously. Shouldn't it be already formatted in the dprint PR?

Reply:
No, I haven't formatted the JSON yet; there are some downsides.

Reply:
Also, this formatting is unrelated to the PR; I would not include it.

Reply:
Actually I didn’t notice that until I opened the PR. I will revert it. I don’t know why it’s being formatted on saving. You might want to review the dprint config.

Reply:
The config works fine on my machine; we don't even have the JSON plugin enabled, so you may want to check what setting you have and if it matches the template.